packages/website/pages/_app.js

```diff
@@ -27,6 +28,16 @@ const App = ({ Component, pageProps }) => {
     <AppProviders authorizationProps={{ ...pageProps }}>
       <Metadata {...pageProps} />
       <RestrictedRoute {...pageProps}>
+        <Script src="https://track.web3.storage/latest.js" data-hostname="web3.storage" data-ignore-pages="/callback" />
```

Comment:
what's the best place to understand what these scripts are doing? it would be helpful to have a comment in here with a pointer for folks who come later and are trying to figure out where scripts come from and how they work